homeassistant/components/google_air_quality/icons.json

```diff
@@ -0,0 +1,18 @@
+{
+  "entity": {
+    "sensor": {
+      "carbon_monoxide": {
+        "default": "mdi:molecule-co"
+      },
+      "nitrogen_dioxide": {
+        "default": "mdi:molecule"
+      },
+      "ozone": {
+        "default": "mdi:molecule"
+      },
+      "sulphur_dioxide": {
+        "default": "mdi:molecule"
+      }
```

Comment:
aren't these all device classes?

Reply:
Yes, they are, but the API delivers other units, than in the device classes allowed.

Reply:
What do they provide?

Reply:
CO is now a device class. 
The API provides ppb for NO2, O3 and SO2. But home assistant requires µg/m³